src/rgw/rgw_sal_motr.cc

```diff
@@ -1073,6 +1073,9 @@ int MotrBucket::list(const DoutPrefixProvider *dpp, ListParams& params, int max,
       rgw_bucket_dir_entry ent;
       auto iter = vals[i].cbegin();
       ent.decode(iter);
+      std::string key_name = ent.key.name + "/null";
```

Comment:
can we rename this to null_obj_ky_name

Reply:
A more appropriate name would be null_ref_key

Reply:
done